clivendor/vendor.json

```diff
@@ -21,6 +21,18 @@
 			"revisionTime": "2015-10-22T06:55:26Z"
 		},
 		{
+			"checksumSHA1": "ulFwBsxjpyu+bk44+LKDl8AKQiA=",
+			"path": "github.com/aryann/difflib",
+			"revision": "e206f873d14a916d3d26c40ab667bca123f365a3",
+			"revisionTime": "2017-07-10T04:42:30Z"
+		},
+		{
+			"checksumSHA1": "AIxWUvzOjHni8IwkElJw+b1IES8=",
+			"path": "github.com/nickbp/mustache",
```

Comment:
Do the CLIs actually use `mustache` or `difflib`? Or is this vendor directory now more general than just CLI?

Reply:
This is the union of both CLI and bootstrap dependencies

Reply:
Should we then change the folder name `clivendor` to `vendor` or `go-vendor`?

Reply:
I now realize that the name `clivendor` isn't quite accurate, I'll update this to be `govendor`, even though that's technically also the name of the tool used to manage vendoring.

Reply:
Actually I'll try `govendor` just to follow the example of e.g. `frameworks/helloworld` of avoiding punctuation